CMakeLists.txt

```diff
@@ -1600,7 +1600,10 @@ if (ENABLE_LEVEL0)
   pkg_check_modules(LEVEL0 REQUIRED level-zero>=1.3)
 
   if(NOT ENABLE_LLVM)
-    message(FATAL_ERROR "Level Zero requires LLVM/Clang")
+    message(FATAL_ERROR "Level Zero requires LLVM/Clang <= 15")
+  endif()
+  if(LLVM_VERSION VERSION_GREATER_EQUAL 16.0)
```

Comment:
Can you add a comment that states what breaks currently (whatever you know about it)?

Reply:
Clang 16 produces bitcode with opaque pointers (despite PoCL passing -no-opaque-pointers). SPIR-V translator seems to be Ok with it, but the Level0 driver crashes inside the compiler.  I'll add a comment.